code/ui/blocks/src/blocks/enhanceSource.ts

```diff
@@ -76,18 +75,14 @@ const extract = (targetId: string, { source, locationsMap }: StorySource) => {
   return extractSource(location, lines);
 };
 
-export const enhanceSource = (story: PreparedStory<any>): Parameters => {
-  const { id, parameters } = story;
-  const { storySource, docs = {} } = parameters;
-  const { transformSource } = docs;
+export const enhanceSource = (story: PreparedStory<any>): string => {
+  const {
+    id,
+    parameters: { storySource },
+  } = story;
 
-  // no input or user has manually overridden the output
-  if (!storySource?.source || docs.source?.code) {
-    return null;
-  }
-
-  const input = extract(id, storySource);
-  const code = transformSource ? transformSource(input, story) : input;
+  // If there's no input, return nothing
+  if (!storySource?.source) return '';
 
-  return { docs: combineParameters(docs, { source: { code } }) };
+  return extract(id, storySource);
 };
```

Comment:
@shilman - so can we drop this file entirely now?

Reply:
I believe so!

Reply:
Reading the code for `enhanceSource` it:

1. Reads `parameters.storySource.source` and `parameters.storySource.locations`
2. Trims the source if locations is set.
3. Returns that.

You are saying that parameter will never be set so we shouldn't bother with any of that?

If so, I'm a little confused about the `getSnippet()` function now, and the `docs.source.type` option. The logic currently is:

1. If `parameters.docs.source.code` is set, use that, OR
2. If we have a dynamic snippet, and type is `DYNAMIC` (or `AUTO` under certain conditions), use that.
3. If type is `CODE` (or `AUTO` otherwise), do the above extraction.

If we remove the extraction, case 3 will lead to... nothing. Is that right?

Reply:
https://github.com/storybookjs/storybook/blob/next/code/lib/csf-tools/src/enrichCsf.test.ts#L37-L43

@tmeasday I take it back. We still get the source code from the same place as before. The difference is that there is no `locationsMap` so that we don't have to apply the extraction logic. We can just use the entire string, so it should simplify things considerably.